### PR TITLE
@veupathdb/components and plotly.js version up

### PR DESCRIPTION
**Pull request by moontrip**
simply version up of @veupathdb/components did not resolve plotly.js version, so I have added it at dependencies. But also realized that in the meantime, plotly.js was upgraded to v2.6.0. For matching the version, I also made a [PR at web-components](https://github.com/VEuPathDB/web-components/pull/242)

### CODE REVIEW

**Comment by moontrip** [comment]
I have also looked at plotly.js's release note but no specifically potential issue was noticed for our cases: https://github.com/plotly/plotly.js/releases/

**Review by bobular — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "io-ts": "^2.2.13",
     "lodash": "^4.17.20",
     "monocle-ts": "^2.3.11",
+    "plotly.js": "^2.5.1",
```
Comment (bobular) [inline package.json:24]: Are we 100% sure we need this?  I'm going to try without...

**Review by bobular — CHANGES_REQUESTED** [review]
I approved, but then I saw the direct dependency for plotly.   Not sure!

File: package.json (hunk 1 above)
Comment (bobular) [reply package.json:24]: I'm stuck too!

If I do `yarn remove plotly.js` then yarn.lock seems to have entries for v2.6.0 but the all important `node_modules/plotly.js/package.json` still says `"version": "1.58.5"`

I tried deleting the node_modules directory and rebuilding but it's the same.

Very strange - need help please @dmfalke @jtlong3rd

Comment (moontrip) [reply package.json:24]: @bobular First of all, hank you for your approval and merge at web-components PR - I just checked emails. As for the plotly.js entry, yes it probably needs to have it at dependencies as it did not automatically upgrade with the web-components. That was the reason why I added it in there. This issue arose sometime last week when Connor noticed this. Such an approach was my solution to resolve.

Comment (dmfalke) [reply package.json:24]: I believe the issue last week was to do with using `yarn link`. That will not necessarily trigger a transitive dependency to be upgraded.

Running `yarn upgrade <packageName>` should cause transitive dependencies to be updated. I will investigate further soon.

Comment (dmfalke) [reply package.json]: I was able to resolve this by deleting `yarn.lock` and running `yarn install` again. I just pushed this change.

**Review by dmfalke — APPROVED** [review]

**Pull request merged**